packages/venia-concept/src/components/CategoryList/categoryList.js

```diff
@@ -0,0 +1,105 @@
+import { Component, createElement } from 'react';
+import { string, number, shape } from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import classify from 'src/classify';
+import defaultClasses from './categoryList.css';
+
+// TODO: get only active categories from graphql when it is ready
+const categoryListQuery = gql`
+    query category($id: Int!) {
+        category(id: $id) {
+            children {
+                name
+                url_key
+                url_path
+                image
+            }
+        }
+    }
+`;
+
+// TODO: get baseUrl from graphql when it is ready
+const baseUrl = 'https://magento-venia.local.pwadev:8000';
```

Comment:
You should not need to hardcode the full domain! Relative paths from root should be fine. Are you working with Luma sample data?

Reply:
Yep, I use magento 2.3-develop + sampledata